src/mrack/transformers/transformer.py

```diff
@@ -72,3 +73,13 @@ def validate_host(self, host):
         provider = host.get("provider")
         if provider and provider not in providers.names:
             raise MetadataError(f"Error: Invalid host provider: {provider}")
+
+    def create_host_requirement(self, host):
+        """Create single input for provisioner."""
+        raise NotImplementedError()
+
+    def create_host_requirements(self):
```

Comment:
nitpick: I would like to see these changes in another commit.

Reply:
Will do.

Reply:
Removed i will create another PR